src/utils/vc/schema.js

```diff
@@ -28,7 +29,7 @@ export async function validateCredentialSchema(credential, schema, context) {
       delete subject[credentialIDField];
     }
 
-    const compacted = await jsonld.compact(subject, context); // eslint-disable-line
+    const compacted = await jsonld.compact(subject, context, { documentLoader: documentLoader || defaultDocumentLoader() }); // eslint-disable-line
```

Comment:
Why wasn't the `documentLoader` required before? This is just to resolve the context URLs, right?

Reply:
yeah to resolve context urls, not sure why it wasnt there before - guess an oversight thats been there a while